raiden/transfer/balance_proof.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+from raiden.encoding.messages import (
+    nonce as nonce_field,
+    transferred_amount as transferred_amount_field,
+)
+
+
+def singing_data(
```

Comment:
A typo in a function name is something I would like you to correct before merging though.

```diff
- def singing
+ def signing
```